runtime/src/launch/slurm-srun/launch-slurm-srun.c

```diff
@@ -240,6 +243,21 @@ static char* chpl_launch_create_command(int argc, char* argv[],
     memStr = memEnv;
   }
 
+  localesPerNode = 1;
+  if (lpnEnv != NULL) {
+    localesPerNode = atoi(lpnEnv);
+    if (localesPerNode <= 0) {
+      char msg[100];
+      snprintf(msg, sizeof(msg), "%s must be > 0.", CHPL_LPN_VAR);
+      chpl_warning(msg, 0, 0);
+      localesPerNode = 1;
+    }
+  }
+
+  if (localesPerNode > numLocales) {
+    localesPerNode = numLocales;
+  }
```

Comment:
What is this adjustment doing?

Reply:
This was for when running something like `CHPL_RT_LOCALES_PER_NODE=2 ./app -nl 1` to give that locale the entire node, but I think we've decided to back that out for now since we're not quite sure what the right default policy is here

Reply:
Removed this adjustment from the code so, for example, if you specify 2 locales per node and run only 1 locale it will only use the cores in its socket.